### DISCUSSION TITLE
temáticas para proyecto

### DISCUSSION THREAD

**Discussion by Idelgado1006** [hackatomik/SISCOx256#55] [Mi Proyecto | Lab]
buenas, en nuestro grupo aún estamos pensando que hacer para nuestro proyecto final, sabemos que la temática es libre.
entre algunos miembros de grupo conversamos y nos interesa la idea de hacer una página web, o quizá un videojuego, o porque no, ambas.
la duda es, en caso de hacer un video juego, (si es que se puede), ¿cómo sería la evaluación? o ¿que se tomaría en cuenta para evaluarlo? esto para saber mas o menos que tipo de enfoque debemos darle al proyecto, gracias!

**Comment by quantumquirkxyz** [comment]
Hola, bueno, por ahora estamos viendo la forma de "evaluar" a las tribus, pero son libres de hacer un videojuegos o una pag web o ambas, si ustedes desean, próximamente estaremos enviando en Teams, la forma en que evaluaremos, por ahora, céntrate en definir muy bien lo que quieres construir con tu tribu.
